static/scripts/ocap.js

```diff
@@ -550,9 +550,17 @@ function processOp (filepath) {
 					groups.addGroup(group);
 				}
 
-				// Create unit and add to entities list
-				var unit = new Unit(startFrameNum, id, name, group, entityJSON.side, (entityJSON.isPlayer == 1), positions, entityJSON.framesFired);
-				entities.add(unit);
+				if (entityJSON.role) {
```

Comment:
I don't think this if is needed, because you can forward this variable all the time, if it not exists its `undefined` and the condition will be false, so no role information is shown.